### ISSUE TITLE
This card cant be used for your payment.please add a different or credit card

### ISSUE THREAD

**Issue by Jason656844** [woocommerce/woocommerce-paypal-payments#1418]
### To Reproduce

1. Go to '…'
2. Click on '…'
3. Scroll down to …'

### Environment

 - WordPress Version
 - WooCommerce Version
 - Plugin Version
 - Browser [e.g. Chrome, Safari] and Version
 - Any other plugins installed

### Additional Details

<details><summary>System status</summary>

  ```
  <!--If applicable, paste the system status here. Please ensure you redact or remove any identifying information. -->
  ```

</details>

**Comment by Jason656844** [comment]
This card cant be used for your payment.please add a different or credit card》
I installed codarab Payment Gateway, which is a PayPal based gateway, but I can't pay by credit card on my website; If I install PayPal payment gateway, the same card can pay

**Comment by InpsydeNiklas** [comment]
Hi @Jason656844 

> I installed codarab Payment Gateway, which is a PayPal based gateway, but I can't pay by credit card on my website; If I install PayPal payment gateway, the same card can pay

If you have any questions relating to third-party PayPal plugins, I suggest reaching out to the respective developer. 

This repository only tracks bugs in the [PayPal Payments plugin](https://woocommerce.com/products/woocommerce-paypal-payments/). If you have any questions or encounter issues with this plugin, please consider [reaching out to the support team](https://woocommerce.com/document/woocommerce-paypal-payments/#get-help) directly for more guidance. Thanks!

**Comment by Jason656844** [comment]
This gateway is based on PayPal, please look at it as a PayPal problem

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Niklas Gutberlet ***@***.***>
发送时间: Monday, June 5, 2023 5:56:36 PM
收件人: woocommerce/woocommerce-paypal-payments ***@***.***>
抄送: Jason656844 ***@***.***>; Mention ***@***.***>
主题: Re: [woocommerce/woocommerce-paypal-payments] This card cant be used for your payment.please add a different or credit card (Issue #1418)


Closed #1418<https://github.com/woocommerce/woocommerce-paypal-payments/issues/1418> as completed.

―
